ui/lib/components/CardTableV2/index.tsx

```diff
@@ -132,10 +134,13 @@ function CardTableV2(props: ICardTableV2Props) {
     }
     newColumns = newColumns.map((c) => ({
       ...c,
+      isResizable: c.isResizable === false ? false : true,
       isSorted: c.key === orderBy,
       isSortedDescending: desc,
       onColumnClick,
+      columnActionsMode: c.columnActionsMode || ColumnActionsMode.disabled,
```

Comment:
disabled is set for non-sortable columns, and for sortable columns, the default is used.

Reply:
Yes, because I found we need to write much `ColumnActionsMode.disabled` more than `ColumnActionsMode.clickable`